torchrl/modules/models/multiagent.py

```diff
@@ -81,48 +88,77 @@ def exec_module(params, *input):
 
         return torch.vmap(exec_module, *args, **kwargs)
 
-    def forward(self, *inputs: Tuple[torch.Tensor]) -> torch.Tensor:
-        if len(inputs) > 1:
-            inputs = torch.cat([*inputs], -1)
+    def _in_dim(self, input: Tuple[...]):
+        # We must define a structure that resembles the input data with the
+        # agent dims in it, smth like (None,) if there is a single input
+        # of (None, (None, None)) for an LSTM
```

Comment:
I do not get, why for an LSTM this is `(None, (None, None))` and not `(None, None, None)`, what is the reason for this nesting?

Reply:
oh got it, i guess it is a torch convention